ApiBundle/Controller/AreaFlexController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\Controller;
+
+use OpenOrchestra\ModelInterface\Event\TemplateFlexEvent;
+use OpenOrchestra\BaseApiBundle\Controller\Annotation as Api;
+use OpenOrchestra\ModelInterface\TemplateFlexEvents;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration as Config;
+use Symfony\Component\HttpFoundation\Response;
+use OpenOrchestra\BaseApiBundle\Controller\BaseController;
+
+/**
+ * Class AreaFlexController
+ *
+ * @Config\Route("area-flex")
+ *
+ * @Api\Serialize
+ */
+class AreaFlexController extends BaseController
+{
+    /**
+     * @param string      $areaId
+     * @param string      $templateId
+     * @param string|null $parentId
+     *
+     * @Config\Route("/{areaId}/delete-in-template/{parentId}/{templateId}", name="open_orchestra_api_area_flex_column_delete_in_template")
+     * @Config\Route("/{areaId}/delete-in-template/{templateId}", name="open_orchestra_api_area_flex_row_delete_in_template")
+     * @Config\Method({"DELETE"})
+     *
+     * @Config\Security("is_granted('ROLE_ACCESS_UPDATE_TEMPLATE_FLEX')")
+     *
+     * @return Response
+     */
+    public function deleteAreaInTemplateAction($areaId, $templateId, $parentId = null)
```

Comment:
The parentId naming is a little ambiguous: can you specify the kind of parent it is ? (ie areaParentId or templateParentId)